src/app/ticker/actions.js

```diff
@@ -31,16 +31,35 @@ export type Action =
 
 // eslint-disable-next-line
 export const register = () => async (dispatch: Function, getState: GetState) => {
-  dispatch(ui.txStart('Submitting token symbol registration...'))
-  try {
-    const details: SymbolDetails = getState().form[formName].values
-    await TickerRegistry.registerTicker(details)
-
-    dispatch(ui.txEnd(null))
-    dispatch(registered())
-  } catch (e) {
-    dispatch(ui.txFailed(e))
-  }
+  dispatch(
+    ui.showModal(
+      'Before you proceed with your Token Symbol Reservation',
+      'Confirmation Required',
+      'Reserve Ticker',
+      'Once you hit &laquo;RESERVE TICKER&raquo;, your Token Symbol ' +
+        'reservation will be sent to the blockchain and will be ' +
+        'immutable.Any change will require that you start the process ' +
+        'over.If you wish to review your information, please select' +
+        '& laquo; CANCEL & raquo;.',
+      'red',
+      'warning--glyph',
```

Comment:
so we don't need prop above and prop here

Reply:
Removed